### ISSUE TITLE
Remove OpenCV Dependency

### ISSUE THREAD

**Issue by RoyiAvital** [zanazakaryaie/fastGaussianBlur#1]
It will be nice to have the project with no OpenCV dependency but reading the image.

**Comment by zanazakaryaie** [comment]
I used OpenCV to load, compare, and store the results. You can use libjpeg or libpng if you just need to load and store the images. [Here](https://github.com/LuaDist/libjpeg/blob/master/example.c) is an example to use libjpeg for reading and writing images

**Comment by RoyiAvital** [comment]
You also use it for the calculation itself:

![image](https://user-images.githubusercontent.com/7347975/118388946-05fbb700-b630-11eb-9785-d28685a4949c.png)

Or am I missing something?

**Comment by zanazakaryaie** [comment]
Yes, that's right. I have explained in the [blog post](http://imrid.net/?p=4173) that using OpenCV's universal intrinsics makes it easier to vectorize the code. But note that this piece of code just serves as a tutorial. If you replace these intrinsics with the scalar version of vertical convolution and pass `-mfpu=neon -mfloat-abi=hard` to GCC, it will autovectorize it for you.
